tests/common/mellanox_data.py

```diff
@@ -26,6 +26,32 @@
         },
         "ports": {
             "number": 32
+        },
+        "thermals": {
+            "cpu_core": {
+                "start": 0,
+                "number": 2
+            },
+            "module": {
+                "start": 1,
+                "number": 32
+            },
+            "psu": {
+                "start": 1,
+                "number": 2
+            },
+            "cpu_pack": {
```

Comment:
This one is repeated here. We'd better think about this dict as a whole. Try to

Reply:
Here "cpu_pack" is not about how many cpu packs on this switch, but how many thermal zones on cpu packs. Although the value is the same, it is harmless to define it here.